L1Trigger/L1CaloTrigger/interface/Phase2L1CaloJetEmulator.h

```diff
@@ -0,0 +1,495 @@
+#ifndef _PHASE_2_L1_CALO_JET_EMULATOR_H_
```

Comment:
This header protection does not follow the style in cms coding:
https://cms-sw.github.io/cms_coding_rules.html#4--technical-coding-rules-1